### PR TITLE
Update csv roles #875

### LINKED ISSUE #875: OLM installed Operator cannot patch resources on OpenShift
**OpenShift versions:** 4.6, 4.7
**Install Namespace:** all
**Operator CSV version:** 2.1.2

**Issue:** infinispan-operator service-account doesn't have authorization to patch `Infinispan CR`

**Steps to reproduce:**
* Install Infinsipan Operator 2.1.2 from OperatorHub to all namespaces
* Create minimal Infinispan CR as shown bellowe

**Infinispan CR**
```
apiVersion: infinispan.org/v1
kind: Infinispan
metadata:
  name: minimal-setup
spec:
  replicas: 1
```

**Logs:**
```
{"level":"info","ts":1614877689.9395955,"logger":"controller_infinispan","msg":"+++++ Reconciling Infinispan. Operator Version: 2.1.1.Final","Request.Namespace":"infinispan","Request.Name":"minimal-setup"}
{"level":"info","ts":1614877689.9396226,"logger":"secret-controller","msg":"Infinispan CR not ready","Reconciling":"Secret","Request.Namespace":"infinispan","Request.Name":"minimal-setup"}
{"level":"info","ts":1614877689.9396226,"logger":"service-controller","msg":"Infinispan CR not ready","Reconciling":"Service,Route,Ingress","Request.Namespace":"infinispan","Request.Name":"minimal-setup"}
{"level":"info","ts":1614877689.9396021,"logger":"config-controller","msg":"Infinispan CR not ready","Reconciling":"ConfigMap","Request.Namespace":"infinispan","Request.Name":"minimal-setup"}
{"level":"info","ts":1614877689.9462056,"logger":"controller_infinispan","msg":"Serving certificate service present. Configuring into Infinispan CR","Request.Namespace":"infinispan","Request.Name":"minimal-setup"}
{"level":"info","ts":1614877689.9490483,"logger":"controller_infinispan","msg":"Error reading the object","Request.Namespace":"infinispan","Request.Name":"minimal-setup"}
{"level":"info","ts":1614877689.949062,"logger":"controller_infinispan","msg":"----- End Reconciling Infinispan.","Request.Namespace":"infinispan","Request.Name":"minimal-setup"}
{"level":"error","ts":1614877689.9490776,"logger":"controller-runtime.controller","msg":"Reconciler error","controller":"infinispan-controller","request":"infinispan/minimal-setup","error":"infinispans.infinispan.org \"minimal-setup\" is forbidden: User \"system:serviceaccount:openshift-operators:infinispan-operator\" cannot patch resource \"infinispans\" in API group \"infinispan.org\" in the namespace \"infinispan\"","stacktrace":"github.com/go-logr/zapr.(*zapLogger).Error\n\t/go/pkg/mod/github.com/go-logr/zapr@v0.1.1/zapr.go:128\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).reconcileHandler\n\t/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.4.0/pkg/internal/controller/controller.go:258\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem\n\t/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.4.0/pkg/internal/controller/controller.go:232\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).worker\n\t/go/pkg/mod/sigs.k8s.io/controller-r...
```

### PR DESCRIPTION
**Pull request by rigazilla**
Some roles are missing in csv
Fixes #875

### CODE REVIEW

File: deploy/olm-catalog/infinispan-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -255,7 +257,7 @@ spec:
                 - watch
                 - create
                 - update
-                - delete
```
Comment (Crumby) [inline deploy/olm-catalog/infinispan-operator.clusterserviceversion.yaml:258]: Won't this cause issues when deleting Infinispans or is `delete` operation included in `patch`?

Comment (dmvolod) [reply deploy/olm-catalog/infinispan-operator.clusterserviceversion.yaml:258]: @Crumby seems to be not, as operator itself never deletes own CR's

Comment (rigazilla) [reply deploy/olm-catalog/infinispan-operator.clusterserviceversion.yaml:258]: yeah, atm is not needed. I've just aligned csv with the role.yaml and clusterrole.yaml file. I'm going to add some code to the codegen target that automatize this step

**Comment by dmvolod** [comment]
Merged, thanks @rigazilla

**Pull request merged**